<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-A9IKKQ4" failures="4" tests="14" name="PetTests" time="22.884" errors="0" timestamp="2025-08-13T15:14:44 IST">
  <testcase classname="api.test.UserTests" name="testPostUser" time="3.285"/>
  <testcase classname="api.test.UserTests" name="testGetUserByName" time="1.426"/>
  <testcase classname="api.test.UserTests" name="testUpdateUserByName" time="2.577"/>
  <testcase classname="api.test.UserTests" name="testDeleteByUserName" time="1.301"/>
  <testcase classname="api.test.UserTests2" name="testPostUser" time="1.193"/>
  <testcase classname="api.test.UserTests2" name="testGetUserByName" time="1.188"/>
  <testcase classname="api.test.UserTests2" name="testUpdateUserByName" time="2.371"/>
  <testcase classname="api.test.UserTests2" name="testDeleteByUserName" time="1.152">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests2.testDeleteByUserName(UserTests2.java:94)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteByUserName -->
  <testcase classname="api.test.DDTests" name="testPostUser" time="1.176"/>
  <testcase classname="api.test.DDTests" name="testPostUser" time="1.178"/>
  <testcase classname="api.test.DDTests" name="testPostUser" time="1.176"/>
  <testcase classname="api.test.DDTests" name="testDeleteUserByName" time="1.149">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.DDTests.testDeleteUserByName(DDTests.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
  <testcase classname="api.test.DDTests" name="testDeleteUserByName" time="1.186">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.DDTests.testDeleteUserByName(DDTests.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
  <testcase classname="api.test.DDTests" name="testDeleteUserByName" time="1.239">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.DDTests.testDeleteUserByName(DDTests.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
</testsuite> <!-- PetTests -->
